db/migrate/20220601190711_change_user_email_not_null.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeUserEmailNotNull < ActiveRecord::Migration[7.0]
+  def up
+    change_column :users, :email, :string, null: false
+  end
+
+  def down
+    change_column :users, :email, :string
+  end
```

Comment:
Can we just define a `change` method instead of `up` and `down`? ActiveRecord will handle the rollback logic automatically for this kind of migrations

Reply:
previously I tried to run a rollback with the `change` and it gave me an error in the console, so I changed by `up` and `down`